scripts/instance.template.env

```diff
@@ -57,10 +57,17 @@ ZWE_CACHING_STORAGE_SIZE=10000
 # specify eviction strategy to be used when the storage size is achieved
 ZWE_CACHING_EVICTION_STRATEGY=reject
 # specify persistent method of caching service
-# possible options: <empty> (will fall back to inMemory), VSAM, redis
-ZWE_CACHING_SERVICE_PERSISTENT=
+# possible options: <empty> (will fall back to inMemory), VSAM, redis, infinispan
+ZWE_CACHING_SERVICE_PERSISTENT=infinispan
```

Comment:
This will change the default persistent from `inMemory` to `infinispan` if that's the intention.

Reply:
Yes it's just for testing, I wanted to check whether there was problem on the install test.